docs/modules/ROOT/pages/index.adoc

```diff
@@ -483,6 +475,7 @@ In each of the above sections, the following changes MUST be applied:
 
 * Any reference to the _claim_ MUST be replaced with the CBOR serialization of the `referenced_assertions` field from the *<<_identity_assertion,identity assertion>>.*
 * Any reference to the _claim generator_ MUST be replaced with the _<<_actor,actor>>_ whose _X.509 certificate_ is being used for this assertion.
+* The validator SHALL maintain a trust list which SHALL be independent from the trust list used to validate claim generator signatures as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_c2pa_signers++[Section 14.4.1, “C2PA Signers.”] The trust list for identity assertion signers need not include the C2PA Trust List.
```

Comment:
Where is the C2PA Trust List specified? (should be good to add a pointer in the spec too)

Reply:
@christianpaquin this is current work of the C2PA Trust List Task Force, which hasn't yet (as far as I know) published any specifications specifics to the trust list. I don't think there's more to link at this time to than §14.4.1, which says there is (or should be) such a trust list. I added another "as defined in that specification" to that section; not sure what more I can do until C2PA publishes more.